operator/Makefile

```diff
@@ -22,10 +22,16 @@ prep-release:
 	kustomize build ./crd > ./output/crd.yaml
 	ls -al ./output
 
-metacontroller/deploy:
+cert-manager/deploy:
+	$(KUBECTL) apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.15.0/cert-manager.yaml
+
+cert-manager/undeploy:
+	-$(KUBECTL_DELETE) apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.15.0/cert-manager.yaml
+
+metacontroller/deploy: cert-manager/deploy
```

Comment:
Should the metacontroller have a target dependency on the cert-manager? I don't think the cert-manager is required to deploy the metacontroller.